src/stores/my-account.ts

```diff
@@ -67,10 +122,24 @@ export const useMyAccount = create<State & Actions>()((set, get) => ({
       const signer = await loginWithSecretKey(secretKey)
       const encodedSecretKey = encodeSecretKey(secretKey)
       address = signer.address
+      const randomPicture = createAvatar(bottts, {
```

Comment:
I think its better to have this not in here, so the avatar generation will be handled by AddressAvatar (related to previous comment in AddressAvatar)

The name also should not be here

Reply:
Moving this out of the store so it can be generated in the avatarAddress component